ckanext/recombinant/i18n/fr/LC_MESSAGES/recombinant.po

```diff
@@ -283,3 +283,7 @@ msgstr "Créer et mettre à jour les données de la feuille de calcul Excel"
 #: ckanext/recombinant/templates/recombinant/snippets/xls_upload.html:32
 #msgid "Submit"
 #msgstr ""
+
+#: ckanext/recombinant/templates/recombinant/snippets/delete.html:20
+#msgid "Please fill out this field."
+#msgstr "Ce champ est obligatoire."
```

Comment:
@$%&!! - No, I imagine they shouldn't be. Dumb cut and paste error - thanks for catching it.